compiler/plugins/input/TOSA/tosa-iree/InputConversion/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ iree_tablegen_library(
 iree_cc_library(
   NAME
     PassHeaders
+  COPTS
+    "-Icompiler/plugins/input/TOSA"
+    "-I$(GENDIR)/compiler/plugins/input/TOSA"
```

Comment:
These are no-ops (apparently/hopefully) in CMake. Could drop them in the bazel_to_cmake conversion